KerbalStuff/blueprints/mods.py

```diff
@@ -127,8 +127,10 @@ def mod(mod_id, mod_name):
         .order_by(FollowEvent.created):
         follower_stats.append(dumb_object(f))
     json_versions = list()
+    size_versions = list()
```

Comment:
Should probably be a dict, since later you reference the size by `v.id`
```suggestion
    size_versions = {}
```

Reply:
Yep, right now it always throws an `IndexError: list assignment index out of range`, because ids start at one, arrays indices don't.

Reply:
That, and you have to populate the list with something for it to have valid indexes at all.